core/classes.py

```diff
@@ -1,6 +1,89 @@
+from logging import Logger
+
 import discord
 from discord.ext import commands
 
+
 class Cog_Extension(commands.Cog):
-       def __init__(self,bot):
-            self.bot = bot
+    def __init__(self, bot: commands.Bot):
+        self.bot = bot
+
+
+class Hybirdcmd_Aliases():
+    def __init__(self, lang, *cmdName: str):
+
+        self.cmdName = list(cmdName)
+        self.aliases = {}
+        self.description = {}
+        self.brief = {}
+
+        for i in cmdName:
```

Comment:
better to use meaningful iterate name instead of placeholder `i`<br/>
it's not c language that you have to use the index number to access data inside, you can just access it directly from the loop